jdaviz/core/loaders/importers/spectrum_list/spectrum_list.vue

```diff
@@ -1,5 +1,37 @@
 <template>
   <v-container>
+    <div v-if="!disable_dropdown && exposures_items.length > 0">
+      <plugin-select
+        :items="exposures_items.map(i => i.label)"
+        :selected.sync="exposures_selected"
+        :show_if_single_entry="true"
+        :multiselect="exposures_multiselect"
+        :search="true"
+        label="Exposure"
+        api_hint="ldr.exposures.selected ="
```

Comment:
API hint should only exist if we're exposing in the API.

```suggestion
        api_hint="ldr.importer.exposures ="
```

Reply:
Done! Marked as resolved.